### ISSUE TITLE
Fix MC failing tests

### ISSUE THREAD

**Issue by dgasmith** [MolSSI-SSS/SSS_2017_QuESt#58]
@dibyendu92  @yohanna123 I have merged both of your MC modules into a single code. Please inspect the result and fix the failing test cases. Please refresh your downstream versions with:

```
git fetch upstream master
git reset --hard upstream/master
git push --force origin master
```

**Comment by dgasmith** [comment]
@dibyendu92 Thanks the update @yohanna123 I think we still need a test of the MC code.

**Comment by yohanna123** [comment]
Hi @dgasmith I am trying to run py.test -v test_mc.py locally. However, I am getting errors.  

`ImportError while importing test module '/home/yohanna/Gits/SSS_2017_QuESt/tests/test_mc.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_mc.py:3: in <module>
    import quest
quest/__init__.py:11: in <module>
    from . import lj
quest/lj.py:3: in <module>
    from . import driver
quest/driver.py:11: in <module>
    from . import mc_module
quest/mc_module.py:6: in <module>
    from core import core
E   ImportError: No module named 'core'`

The files are updated in my fork. Thank you.

**Comment by loriab** [comment]
It's just not finding `core.so`. Provided you've compiled it, try running pytest while adding the dir containing `core.so` to your :envvar:`PYTHONPATH`. That is, `PYTHONPATH=/home/yohanna/Gits/SSS_2017_QuESt/quest/core pytest -v test_mc.py`

**Comment by dgasmith** [comment]
Agreed, I think you need to compile it via `python setup.py cmake`.

**Comment by dgasmith** [comment]
@yohanna123 I patched up your PR. Please give it a look when you have a chance.

**Comment by yohanna123** [comment]
Thanks @dgasmith. It looks good.  In test_mc.py, I changed coordinates to random coordinates. Was this what you meant by using the random number generator?

**Comment by yohanna123** [comment]
or should I use the mc function on random coordinates..and test if the energetic value is reasonable after simulation.

**Comment by dgasmith** [comment]
If we were to use a random set of data we would probably want to prevent a fully random test by setting the seed. Fortunately, we had nice test data to use!

I think I was referring to using a random seed when it comes to testing an actual MC run. 

-Daniel Smith

**Comment by dgasmith** [comment]
To be explicated we do not test the [MC function](https://github.com/MolSSI-SSS/SSS_2017_QuESt/blob/master/quest/mc_module.py#L97). This can be a deterministic procedure if we start from the same random seed each time.
